language/e2e-tests/src/tests/transaction_builder.rs

```diff
@@ -66,40 +66,49 @@ fn register_preburn_burn() {
 }
 
 #[test]
-fn approved_payment() {
+fn travel_rule_payment() {
     let mut executor = FakeExecutor::from_genesis_file();
-    // account that will receive the approved payment
+    let association = Account::new_association();
+    // account that will receive the travel rule payment
     let payment_receiver = {
         let data = AccountData::new(1_000_000, 0);
         executor.add_account_data(&data);
         data.into_account()
     };
-    // account that will send the approved payment
+    // account that will send the travel rule payment
     let payment_sender = {
         let data = AccountData::new(1_000_000, 0);
         executor.add_account_data(&data);
         data.into_account()
     };
 
-    // Register the receiver account
     let mut keygen = KeyGen::from_seed([9u8; 32]);
     let (private_key, public_key) = keygen.generate_keypair();
-    executor.execute_and_apply(payment_receiver.signed_script_txn(
-        encode_register_approved_payment_script(public_key.to_bytes().to_vec()),
-        0,
+    // rotate the recipient VASP's compliance public key to a known value
```

Comment:
@tzakian: could you help me figure out how to make this work in the new world where VASP is the default account type? I get

```
thread 'tests::transaction_builder::travel_rule_payment' panicked at 'transaction failed with VMStatus { major_status: ABORTED, sub_status: Some(2001), message: Some("0x00000000000000000000000000000000::AccountType::update at offset 28") }', language/e2e-tests/src/executor.rs:211:17
```

I also tried starting by having `payment_receiver` register a `RootVASP` resource, but then I get a failure because the resource already exists.

Reply:
Ah...so this is unfortunate. The VASP where the compliance key is being rotated can't be created as a VASP (there's certain metadata that needs to be published as well). You'll need to do the flow of:
1. Create as an empty account
2. Have that empty account apply to be a VASP
3. Have the association grant the VASP
4. Then you can have the association rotate this key

Reply:
Got it. Is it possible to create an empty account in E2E tests? I know how to do it for functional, but I can't find the right API in Rust-land.

Reply:
Agh! I messed this up. In `account.rs` there is a function called `new_empty`. It uses the wrong AccountTypeSpecifier at the moment (Vasp). If you change it to `AccountTypeSpecifier::Empty` you should be able to create an empty account in Rust-land.

Reply:
Thanks! That fixed it.